packages/lib/defaultEvents.ts

```diff
@@ -33,6 +22,31 @@ type UsernameSlugLinkProps = {
   slug: string;
 };
 
+const user = {
```

Comment:
Can we do this instead of casting `as User` where is used?
```suggestion
const user: User = {
```